spinup.sh

```diff
@@ -4,4 +4,4 @@
 
 imageName="dockerdev"
 
-docker run -it -v "$(pwd)":/home $imageName bash
+docker run -it -v "$(pwd)":/home --env-file env.list $imageName bash
```

Comment:
Maybe set the environment variables inside the container via a run script. I think it would be easier for a user to edit config settings in a run script instead of possibly stopping the container to edit the env.list and spinning up the image again.